.github/ISSUE_TEMPLATE/feature-request.md

```
@@ -0,0 +1,28 @@
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Approach to be followed (optional)**
+A clear and concise description of approach to be followed.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+**Potential internship candidates**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)